UndertaleModLib/Models/UndertaleRoom.cs

```diff
@@ -838,9 +888,14 @@ public override string ToString()
         public class Tile : UndertaleObject, RoomObject, INotifyPropertyChanged
         {
             /// <summary>
-            /// Whether this tile is from an asset layer. Game Maker Studio: 2 exclusive.
+            /// Whether this tile is from an asset layer.<br/>
+            /// Equals true if it's Game Maker Studio: 2.
```

Comment:
Is this true in *all* GMS2 games?

Reply:
Yes.
`spriteMode = reader.undertaleData.GeneralInfo.Major >= 2;`